app/javascript/packs/articlePage.jsx

```diff
@@ -47,7 +48,18 @@ if (shareDropdownButton.dataset.initialized !== 'true') {
     document
       .querySelectorAll('#article-show-more-dropdown [href]')
       .forEach((link) => link.addEventListener('click', closeDropdown));
+
+    // Add Ahoy stats for dropdown share options
+    const container = document.getElementsByClassName(
+      'crayons-article-actions__inner',
+    )[0];
+    container?.querySelectorAll('a.dropdown-link-row').forEach((link) => {
+      link.addEventListener('click', (event) => {
+        ahoy.track('Post Dropdown', { option: event.target.text.trim() });
+      });
+    });
```

Comment:
We already grab and loop through all of these links on line 48, so I think it would be more efficient to add these event listeners in that same loop, e.g:

```js
 // We want to close the dropdown on link select (since they open in a new tab)
 // We also have Ahoy stats for dropdown share options
    document
      .querySelectorAll('#article-show-more-dropdown [href]')
      .forEach((link) => {
        link.addEventListener('click', closeDropdown);
        ahoy.track('Post Dropdown', { option: event.target.text.trim() });
      });
```